src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="image card__image"
+        src="./images/imac.jpeg"
+        title="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+        alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+      >
+
+      <h2 class="title card__title">
+        <a
+          class="link title__link"
+          data-qa="hover"
+          href="#">APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
+        </a>
+      </h2>
+
+      <p class="product-code card__product-code">Product code: 195434</p>
+
+      <div class="stars card__stars">
+        <div class="stars__stars stars__stars--4">
```

Comment:
Do you really need stars__stars? Can't it be just stars without an extra container above?

Reply:
container <stars__stars> allows you to position all <star> and <stars__reviews> among themselves inside <stars card__stars>

Reply:
container <stars__stars> allows you to position all <star> and <stars__reviews> among themselves inside <stars card__stars>

Reply:
inside <stars card__stars>

Reply:
<stars. card__stars>

Reply:
inside <stars card__stars>

Reply:
inside <stars. card__stars>